### ISSUE TITLE
[Question] Citylights not working

### ISSUE THREAD

**Issue by jrodrigv** [PhineasFreak/RSSVE#23]
Hi @PhineasFreak ,

I have been testing latest RSSVE, EVE and Scatterer with KSP 1.6 and it seems I can't get the night lights to work.

Do you know if there is something wrong related with the citylights stuff?

**Comment by PhineasFreak** [comment]
First of all, how are you testing with KSP 1.6? I do not even have a KSP 1.6 install active yet...

Now on the bug: as you understand i cannot guess what is going on, especially on an untried KSP version. As a result **[i will direct you to the relative Wiki entry](https://github.com/PhineasFreak/RSSVE/wiki/Q&A#help-after-installing-rssve-i-have-the-x-y-z-problem)**. Keep in mind that if it is a KSP 1.6 bug then i will not be able to fix it.

**Comment by jrodrigv** [comment]
Well, it is just a question to know if this a known bug or if you haven't ever seen the night lights not working.

It might not be a RSSVE problem but also related with EVE.

**Comment by PhineasFreak** [comment]
No, i have never seen that. If you are using version 1.3.1-7 then it may be related to the fact that RSSVE locks itself if the KSP version does not match, so any Difficulty Page settings (including toggling the City Lights) are probably not going to work.

**Comment by jrodrigv** [comment]
Thanks @PhineasFreak . I have recompiled RSSVE  just in case modifing the checker to avoid locking but it seems that is not the problem.

I will do more testing playing with the configs to see if I can find the root cause.

**Comment by PhineasFreak** [comment]
I also created a new 1.6.1 dev environment for testing but when i checked the lights did show up (both in flight and map views). Can you attach a log file here so that i can take a look?

**Comment by jrodrigv** [comment]
uhmm intereristing then I think it might be related with the versions that I'm using for EVE and Scatterer ( I recompiled their branches for KSP 1.6).

Which versions did you use or branch when setting the test environment?

I have attached a KSP.log

[KSP.log](https://github.com/PhineasFreak/RSSVE/files/2775500/KSP.log)

**Comment by PhineasFreak** [comment]
**Mod List:**

* DistantObject v1.9.1.14505
* EVE v1.4.2.2
* HyperEdit v1.5.8.0
* Kopernicus v1.6.1-1
* ModularFlightIntegrator v1.2.6.0
* ModuleManager v3.1.3.0
* Real Solar System v0.15.0.0
* Scatterer v0.04
* SunFixer v1.0.1.0
* TextureReplacer v3.6.2.43173

What i noticed from your log that does not appear on mine:

```
[LOG 12:57:05.547] Ghassen camera GalaxyCamera
[LOG 12:57:05.547] Ghassen camera cullingmask262144
[LOG 12:57:05.547] Ghassen camera nearplane0.1
[LOG 12:57:05.547] Ghassen camera farplane20
[LOG 12:57:05.547] Ghassen camera SkySphere Cam
[LOG 12:57:05.547] Ghassen camera cullingmask262144
[LOG 12:57:05.547] Ghassen camera nearplane0.01
[LOG 12:57:05.547] Ghassen camera farplane6
[LOG 12:57:05.547] Ghassen camera Landscape Camera
[LOG 12:57:05.547] Ghassen camera cullingmask32769
[LOG 12:57:05.547] Ghassen camera nearplane0.3
[LOG 12:57:05.547] Ghassen camera farplane200000
[LOG 12:57:05.547] Ghassen camera CanvasCamera
[LOG 12:57:05.547] Ghassen camera cullingmask32
[LOG 12:57:05.547] Ghassen camera nearplane0.3
[LOG 12:57:05.547] Ghassen camera farplane1000
[LOG 12:57:05.547] Ghassen camera UIMainCamera
[LOG 12:57:05.547] Ghassen camera cullingmask33554464
[LOG 12:57:05.547] Ghassen camera nearplane-1
[LOG 12:57:05.547] Ghassen camera farplane1000
[LOG 12:57:05.547] Ghassen camera UIVectorCamera
[LOG 12:57:05.547] Ghassen camera cullingmask4096
[LOG 12:57:05.547] Ghassen camera nearplane0.1
[LOG 12:57:05.547] Ghassen camera farplane1000
.
.
.
[ERR 12:57:05.554] Invalid antiAliasing value (must be at least 1)
```

I searched the Scatterer repository but i did not manage to find anything that creates these log entries. What version of Scatterer are you using?

**Comment by jrodrigv** [comment]
@PhineasFreak I'm using the Scatterer dev branch compiled. I haven't recompiled the shaders though. 

Regarding the antialiasing I'm going to try to disable a custom Nvidia profile that I have for KSP.

I have also tested using dx11 and dx9 but the outcome is the same regarding the city lights.
